tests/src/Mooc/Courses/Application/Update/RenameCourseCommandMother.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace CodelyTv\Tests\Mooc\Courses\Application\Update;
+
+use CodelyTv\Mooc\Courses\Application\Update\RenameCourseCommand;
+use CodelyTv\Mooc\Courses\Domain\Course;
+use CodelyTv\Mooc\Courses\Domain\CourseName;
+use CodelyTv\Mooc\Shared\Domain\Course\CourseId;
+use CodelyTv\Tests\Mooc\Courses\Domain\CourseNameMother;
+
+final class RenameCourseCommandMother
+{
+    public static function create(CourseId $id, CourseName $name): RenameCourseCommand
+    {
+        return new RenameCourseCommand($id->value(), $name->value());
+    }
+
+    public static function fromCourse(Course $course): RenameCourseCommand
+    {
+        return self::create($course->id(), CourseNameMother::random());
+    }
```

Comment:
If it's `fromCourse` I understand it's usings all its properties, but here we are generating a random name.

Or:
 1. Use the `$course` name
 2. Change `fromCourse` to `withId` and pass only the `CourseId` 😊